zcash_client_backend/src/fees/common.rs

```diff
@@ -240,12 +256,29 @@ where
     // Note that using the `DustAction::AddDustToFee` policy inherently leaks
     // more information.
 
+    let transparent_input_sizes = transparent_inputs
+        .iter()
+        .map(|i| i.serialized_size())
+        .chain(
+            ephemeral_input_amounts
+                .iter()
+                .map(|_| InputSize::STANDARD_P2PKH),
```

Comment:
`ephemeral_input_amounts` only supports P2PKH inputs. I will document that. In any case, it doesn't matter *here* because using `InputSize::STANDARD_P2PKH` would just overestimate the fee, which is fine when we're creating a transaction.